source/pdo_sqlsrv/pdo_util.cpp

```diff
@@ -56,8 +56,8 @@ pdo_error PDO_ERRORS[] = {
     
     {
         SQLSRV_ERROR_DRIVER_NOT_INSTALLED,
-        { IMSSP, (SQLCHAR*) "This extension requires the Microsoft ODBC Driver 13 for SQL Server to "
-        "communicate with SQL Server. Access the following URL to download the ODBC Driver 13 for SQL Server "
+        { IMSSP, (SQLCHAR*) "This extension requires the Microsoft ODBC Driver 13 or 11 for SQL Server to "
+        "communicate with SQL Server. Access the following URL to download the ODBC Driver for SQL Server "
```

Comment:
This message implies ODBC 17 is not included... it looks like this error is only thrown when the user does not request encryption so it's not strictly needed to mention 17, but it still seems odd to leave 17 out. This error message would have to change anyway once 17 is officially released, so maybe we should change it here now to include 17? Though we might need a different link for 17, and then we'd have to remove that link when 17 is released anyway, so it's up to you...